QQE-378.md

```diff
@@ -0,0 +1,44 @@
+# QQE-378 - HTTP module / REST use-cases - Development topic compression Brotli4j
+## Links
+Jira : [QQE-378](https://issues.redhat.com/browse/QQE-378)
+
+PR:  TODO
+## Scope of the testing
+ - Integration of the Brotli4J compression library with Quarkus, particularly in the context of REST endpoints using RestEasy Reactive.
+ - The test coverage needs to be defined for both JVM and native mode.
+ - To use Brotli4J compression we will need a custom Server configuration
+   * Utilize Vert.x http server options  to enable compression support.
```

Comment:
You basically want to set decompression supported https://github.com/eclipse-vertx/vert.x/blob/master/src/main/asciidoc/http.adoc#handling-compressed-body but there [quarkus.http.enable-decompression](https://quarkus.io/guides/all-config#quarkus-vertx-http_quarkus-http-enable-decompression) and same for a management interface. I thin kyou should use them instead.

Reply:
Well, quarkus options like `quarkus.http.enable-compression` and `quarkus.http.enable-decompression` works by default for gzip so for brotli4J we need both options enabled and use Vert.x HTTP server options in some way (as you mentioned above) or directly the Brotli4J library API.